internal/backend/remote-state/s3/validate.go

```diff
@@ -12,6 +12,7 @@ import (
 const (
 	multiRegionKeyIdPattern = `mrk-[a-f0-9]{32}`
 	uuidRegexPattern        = `[a-f0-9]{8}-[a-f0-9]{4}-[1-5][a-f0-9]{3}-[ab89][a-f0-9]{3}-[a-f0-9]{12}`
+	aliasRegexPattern       = `alias/(.*)`
```

Comment:
```suggestion
	aliasRegexPattern       = `^alias/[a-zA-Z0-9/_-]+$`
```

I guess this should match https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-kms-alias.html#aws-resource-kms-alias-properties

Reply:
The match filter for the characters is fine, but because the anchors are already in the code this would break it. I'll add the change to the filter though

Reply:
I've tried several different iterations of the above and they all break the tests. I'm going to leave it as is for now, unless @marcinwyszynski would prefer it gets fixed.

I've tried the above as well as `alias\/[a-zA-Z0-9\/\-\_]+`

Reply:
The anchors can't be part of this `const` because it's part of an expression that already has them. Curious though what breaks when you replace `(.*)` with `[a-zA-Z0-9/_-]+`?

Reply:
Added this change in a different commit, as i don't know if i can edit a suggestion.

Reply:
Ok, I fixed the issue. I needed to add `()` around the regex to define it as a capture group for the one match look up.